HCIDOCS-474-OCI: AI installation on OCI bare metal (#82685)
openshift/openshift-docs: modules/provision-oci-infrastructure-ocp-cluster.adoc

[id="provision-oci-infrastructure-ocp-cluster_{context}"]
= Provisioning {oci} infrastructure for your cluster

> When using the {ai-full} to create details for your {product-title} cluster, you now specify these details in a Terraform stack. A stack is an {oci} feature that automates the provisioning of all necessary {oci} infrastructure resources that are required for installing an {product-title} cluster on {oci}.

Reviewer: ```suggestion
When using the {ai-full} to create details for your {product-title} cluster, you specify these details in a Terraform stack. A stack is an {oci} feature that automates the provisioning of all necessary {oci} infrastructure resources that are required for installing an {product-title} cluster on {oci}.
```

.Prerequisites

> * You downloaded the discovery ISO image to a local directory. For details, see "Using the {ai-full} to generate an {oci}-compatible discovery ISO image" in this chapter.
> * You downloaded the Terraform stack template to a local directory. For details, see "Preparing the OCI environment" in this chapter.

Reviewer: Want to avoid self-reference, but also due to the modularity of content and the published output being subject to change, this could become an untrue statement
```suggestion
* You downloaded the discovery ISO image to a local directory. For details, see "Using the {ai-full} to generate an {oci}-compatible discovery ISO image".
* You downloaded the Terraform stack template to a local directory. For details, see "Preparing the OCI environment".
```

.Procedure

. Log in to your link:https://cloud.oracle.com/a/[{oci-first-no-rt}] account.

> . Upload the discovery ISO image from your local drive to the new object storage bucket you created. For the full procedure, see link:https://docs.oracle.com/en-us/iaas/Content/Object/Tasks/managingobjects_topic-To_upload_objects_to_a_bucket.htm[Uploading an Object Storage Object to a Bucket]. 

Reviewer: All the Oracle docs need (Oracle documentation) tags

> . Locate the uploaded discovery ISO, and do the following:

Reviewer: ```suggestion
. Locate the uploaded discovery ISO, and complete the following steps:
```
